static/src/stylesheets/module/email/_form.scss

```diff
@@ -163,6 +163,28 @@
     @include fs-textSans(1, $size-only: true);
 }
 
+.email-sub--close {
+    background: 0 none;
+    border: 0 none;
+    position: absolute;
+    top: 0;
+    right: 0;
+}
+
+.email-sub--hidden {
+    @include u-h();
+}
+
+.email-sub--close-icon svg {
+    fill: darken(guss-colour(neutral-2, $pasteup-palette), 10%);
```

Comment:
this can just be `colour(neutral-2)`.

guss is a bit crazy, it's something we'll tidy up